src/otg/datasource/ukbiobank/study_index.py

```diff
@@ -49,7 +49,7 @@ def from_source(
         Returns:
             UKBiobankStudyIndex: Annotated UKBiobank study table.
         """
-        return StudyIndex(
+        return UKBiobankStudyIndex(
```

Comment:
I don't really understand this. I understand we have a class to generate UKBB study index. However there's no ukbb study index datatype. When saying `UKBiobankStudyIndex.get_schema()` it's just `StudyIndex.get_schema()` because of the inheritance.

Reply:
I think it's the same comment as the above, but let me know if it's not clear